datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -327,11 +325,10 @@ pub async fn from_substrait_extended_expr(
     })
 }
 
-/// parse projection
-pub fn extract_projection(
-    t: LogicalPlan,
+pub fn apply_projection(
```

Comment:
minor: if we're going to rename this, I might would `apply_masking` as that better corresponds to the fact that this is applying MaskExpressions specifically, and also makes it clearer that this isn't related to creating `Projects` / `Projections`.